cap-ci/cap-pre-release-upgrades.yaml

```diff
@@ -0,0 +1,61 @@
+workertags:
+  # caasp4: suse-internal
+
+jobs:
+  deploy-k8s: true
+  pre-upgrade-deploy-kubecf: true
+  pre-upgrade-smoke-tests: true
+  deploy-stratos: true
+  upgrade-kubecf: true
+  post-upgrade-smoke-tests: true
+  cf-acceptance-tests-brain: true
+  sync-integration-tests: true
+  cf-acceptance-tests: true
+  destroy-kubecf: true
+jobs_ordered:
+- deploy-k8s
+- pre-upgrade-deploy-kubecf
+- pre-upgrade-smoke-tests
+- deploy-stratos
```

Comment:
I think deploy-stratos should happen after the upgrade, as we're currently not doing any validation that stratos will be working after the upgrade, or really doing any stratos validation in general, other than verifying deployment works

Reply:
yes, its true that all we are doing in installing stratos, but any customer who will upgrade will have stratos already installed. We can always expand upgrade verification criteria later to do more with this.
I don't see the benefit of installing stratos post-upgrade as such stratos testing is already covered by non-upgrade CIs (cap release and cap pre release), so there is no point in repeating the test in upgrade CIs